### PR TITLE
Correct check for qunit 2

### PR DESCRIPTION
**Pull request by shayaneumar**

### CODE REVIEW

File: steal-qunit.js [hunk 1]
```diff
@@ -111,8 +111,8 @@ define([
 		if (window.Testee && window.Testee.init) {
 			Testee.init();
 		}
-		
-		if(typeof QUnit.start === "function") {
```
Comment (shayaneumar) [inline steal-qunit.js:115]: This is not the right check for qunit 2, this is vailable in both 1x and 2x
image

http://api.qunitjs.com/QUnit/start

File: steal-qunit.js [hunk 2]
```diff
@@ -111,8 +111,8 @@ define([
...
+
+		if(QUnit.assert && QUnit.assert.ok && QUnit.assert.async) {
```
Comment (matthewp) [inline steal-qunit.js]: This check is truthy in 1.x as well though.  Why not just check the version directly?

```js
var qunitVersion = +QUnit.version.split('.')[0];
if(qunitVersion >= 2) {
  QUnit.start();
} else {
  QUnit.load();
}
```

Comment (shayaneumar) [reply steal-qunit.js]: Yes we can do that, we need to just make sure to add the check for the version in which functionality is changed. I am not sure if that is version 2.

By going through the change log I have found this. not sure what does it mean.
https://github.com/qunitjs/qunit/blob/2.3.2/History.md#211--2017-01-05

image

Comment (shayaneumar) [reply steal-qunit.js]: @matthewp waiting on your input on this on how to proceed.

Comment (matthewp) [reply steal-qunit.js]: QUnit.load() works in all 1.x versions. I would just keep it simple and do the check for 2 as the major version.

File: steal-qunit.js [hunk 3]
```diff
@@ -111,12 +111,14 @@ define([
 		if (window.Testee && window.Testee.init) {
 			Testee.init();
 		}
-		
-		if(typeof QUnit.start === "function") {
```
Comment (shayaneumar) [inline steal-qunit.js:115]: @matthewp Done.

**Comment by shayaneumar** [comment]
@matthewp Made the suggested code changes.

**Comment by matthewp** [comment]
Looks good, thanks!

**Comment by matthewp** [comment]
This change was released in 1.1.0-beta.1

**Pull request merged**